src/luascript.cpp

```diff
@@ -11994,6 +11990,141 @@ int LuaScriptInterface::luaPartySetSharedExperience(lua_State* L)
 	return 1;
 }
 
+// Spells
+int LuaScriptInterface::luaSpellCreate(lua_State* L)
+{
+	// Spell(words, name or id)
+	InstantSpell* spell;
+	if (isNumber(L, 2)) {
+		spell = g_spells->getInstantSpellById(getNumber<uint32_t>(L, 2));
+	} else if (isString(L, 2)) {
```

Comment:
I think you don't need this, just treat the second argument as only number or string. If nil, `isString` will return and empty string that will not match any spell and skip.